CMakeLists.txt

```diff
@@ -58,6 +63,15 @@ catkin_package(
     ${cmake_files}
 )
 
+##########################################
+# Install the interesting resource files #
+##########################################
+configure_file(
+    ${MPI_CMAKE_MODULES_RESOURCES_DIR}/_clang-format
```

Comment:
why the underscore ?

Reply:
this is to differentiate form the file that clang-format look at: .clang-format
This underscore replace the "."

Reply:
It is actually even a valid name for the config file (clang-format is looking for both `.clang-format` and `_clang-format`).